core/src/main/java/org/apache/accumulo/core/trace/TraceUtil.java

```diff
@@ -56,97 +52,32 @@
 
   private static final String SPAN_FORMAT = "%s::%s";
 
-  private static Tracer instance = null;
-  private static boolean tracing = false;
+  private static volatile boolean enabled = true;
```

Comment:
Leaving this on by default means client applications will default to using GlobalOpenTelemetry.get(), and can configure OpenTelemetry (or not) at their discretion. This PR switches this to false by default for server processes that load from the server config file, because the default value of the config option is false. The shell also switches this to false by default, so users can rely on the `trace on` / `trace off` command to toggle it. For the shell, it would also still need to be configured via the class path / autoconfigure (or javaagent) and config properties like other clients.